x-pack/plugins/observability/public/pages/alert_details/types.ts

```diff
@@ -5,8 +5,19 @@
  * 2.0.
  */
 
-export interface PageHeaderProps {
-  alert: unknown;
+export interface PageTitleProps {
+  alert: Alert;
+}
+
+export interface Alert {
```

Comment:
Don't we have an alert interface already? CC @simianhacker

Reply:
@CoenWarmer, no need to create a new type. You can get it from 
`import { Alert } from '@kbn/alerting-plugin/server';`

Reply:
What @fkanout said 😉